### ISSUE TITLE
Parse nyc output

### ISSUE THREAD

**Issue by prashaantt** [prashaantt/node-tdd#2]
Istanbul is being re-written as [nyc](https://github.com/istanbuljs/nyc), whose default output is currently not being parsed by node-tdd.

**Comment by nicosommi** [comment]
@prashaantt  It works for me, are you using the text-summary reporter?
(In the Details for the package it says that is a known issue/limitation)

**Comment by stherrienaspnet** [comment]
Hello Prashaant, do you have plan to support new nyc output in the future?

**Comment by prashaantt** [comment]
Yes, I hope to get to this someday, although the text-summary option works pretty well already.

**Comment by kbrandwijk** [comment]
Would you accept a PR that includes:
- Setting the path to the nyc report
- Setting nyc report format (support all nyc formats)
- Watch report file to update coverage details

**Comment by prashaantt** [comment]
Sure, that would be very nice @kbrandwijk! I had plans to do something similar but haven't been able to get around it yet.

**Comment by kbrandwijk** [comment]
I will create a PR for this next week then. I almost have everything necessary for it running locally, but I don't have any time this week to wrap it up.
